src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue29092Shell.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue29092Shell : _IssuesUITest
+{
+	public Issue29092Shell(TestDevice testDevice) : base(testDevice)
+	{
+	}
+
+	public override string Issue => "Shell - Auto Resize chrome icons on iOS to make it more consistent with other platforms - hamburger icon";
+
+	[Test]
+	[Category(UITestCategories.Shell)]
+	public void ShellFlyoutIconShouldAutoscale()
+	{
+		App.WaitForElement("HelloWorldLabel");
+		VerifyScreenshot();
```

Comment:
Pending snapshots already available in the latest build.

![image](https://github.com/user-attachments/assets/719b8551-5872-4860-a705-a1388f47f2f8)